### ISSUE TITLE
Enable temp features to be clicked on and show minimal pop up

### ISSUE THREAD

**Issue by jessnoo** [Esri/solutions-webappbuilder-widgets#994]
if one of the records gets an unexpected valid geocode, it might be nice to enable the user to be able to click on the feature in the map and see the "key" string -- whatever the string is that would show in the review list -- not all the info, but at least some identifying information to assist with locating it.

for example -- i expected all these records to be in maryland -- so why is one showing up in boston? which one is it it?  i have to cycle through all the "good" items to find out ... 

![image](https://user-images.githubusercontent.com/6181441/32971827-f81db7c0-cba3-11e7-8590-49e74d691838.png)

**Comment by jmhauck** [comment]
@NikkiGolding ready to be verified. This is related to #1011 also. I believe the problem identified here is that popups were not enabled at the right time. If you had not started to review the individual features through the widget the popups would not show if you clicked on one of the features in the map. 

I set the popup to show the fields that are a part of the field information the user has a chance to update through the widget workflows and not show any of the fields we add to these temp features as observed in the related issue. If we would like to refine further what should be displayed I figured that other issue would be the place to do that.
